src/components/ReadingIndicator.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react'
+
+class ReadingIndicator extends PureComponent {
+  scrollHeight = 0
+  state = { amount: 0, }
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.scrolling)
+    this.scrollHeight = Math.max(
+      document.body.scrollHeight,
+      document.body.offsetHeight,
+      document.documentElement.clientHeight,
+      document.documentElement.scrollHeight,
+      document.documentElement.offsetHeight,
+    ) - window.innerHeight
+  }
+
+  componentWillUnmount = () => window.addEventListener('scroll', this.scrolling)
```

Comment:
```suggestion
  componentWillUnmount = () => window.removeEventListener('scroll', this.scrolling)
```
I think this should be removeEventListener